src/loaders/ncbi_downloader/ncbi_downloader.py

```diff
@@ -1,74 +1,76 @@
 """
-PROTOTYPE - Download genome files from NCBI FTP server.
-
-usage: ncbi_downloader.py [-h] --download_file_ext DOWNLOAD_FILE_EXT [DOWNLOAD_FILE_EXT ...] --release_ver
-                          RELEASE_VER [--root_dir ROOT_DIR] [--source SOURCE] [--threads THREADS]
-                          [--overwrite]
+usage: ncbi_downloader.py [-h] --download_file_ext DOWNLOAD_FILE_EXT [DOWNLOAD_FILE_EXT ...] --release_ver {202,207,214,80,83,86,89,95}
+                          [--kbase_collection {GTDB}] [--root_dir ROOT_DIR] [--source {NCBI}] [--threads THREADS] [--overwrite]
                           [--exclude_name_substring EXCLUDE_NAME_SUBSTRING [EXCLUDE_NAME_SUBSTRING ...]]
 
+PROTOTYPE - Download genome files from NCBI FTP server.
+
 options:
   -h, --help            show this help message and exit
 
 required named arguments:
   --download_file_ext DOWNLOAD_FILE_EXT [DOWNLOAD_FILE_EXT ...]
```

Comment:
we should probably make `genomic.fna.gz` default extension. I think `xxx.genomic.fna.gz` the only file we want to download from NCBI.

Reply:
I'm assuming at some point we'll want to download gbk or gff so we can upload genomes as well as assemblies